src/Logger.js

```diff
@@ -154,6 +137,12 @@ class Logger {
         }
         return Promise.resolve('Required sessionId not set');
     }
+
+    info(log, state) { this.report('info', log, state); }
```

Comment:
now given this the ```extraParams``` added in the session will also be sent every time by default.
Can we not modify report function to take ```extraParams``` as an argument too. If it is not there then it can fallback to the session's global extraParams.